### PR TITLE
Save limitQuantityPerCheckout in site settings form

### PR DESCRIPTION
**Pull request by Cloud11PL**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

**Review by witoszekdev — APPROVED** [review]

File: src/siteSettings/views/index.tsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ export const SiteSettings: React.FC<SiteSettingsProps> = () => {
           status: "success",
           text: intl.formatMessage(commonMessages.savedChanges),
         });
+        siteSettings.refetch();
```
Comment (poulch) [inline src/siteSettings/views/index.tsx:39]: question: Do we need this refetch?

File: playwright/pages/siteSettingsPage.ts [hunk 2]
```diff
@@ -58,9 +63,11 @@ export class SiteSettingsPage extends BasePage {
     await this.addressLine2Input.fill(addressLine2);
     await this.city.fill(city);
     await this.countryInput.click();
-    await this.autocompleteDropdown.getByText(country, { exact: true }).click();
+    await this.autocompleteDropdownCountry.getByText(country, { exact: true }).click();
+    await this.autocompleteDropdownCountry.blur();
     await this.countryAreaDropdown.fill(countryArea);
-    await this.autocompleteDropdown.getByText(countryArea, { exact: true }).click();
+    await this.autocompleteDropdownCountryArea.getByText(countryArea, { exact: true }).click();
+    await this.autocompleteDropdownCountryArea.blur();
```
Comment (poulch) [inline playwright/pages/siteSettingsPage.ts:70]: question: why did you change those selectors?

**Review by poulch — APPROVED** [review]

File: playwright/pages/siteSettingsPage.ts (hunk 2 above)
Comment (Cloud11PL) [reply playwright/pages/siteSettingsPage.ts:70]: Before this change there was a single selector for two dropdowns implying that all dropdowns have one id. This test was failing because Combobox doesn't pass test id in a way this test expected it to. I decided to make separate selectors for these two dropdowns and pass id so that we can select it using 'data-portal-for' - https://github.com/saleor/macaw-ui/commit/65a8f3f0a7e56e7f5a4b9ee2e0fd391f5e144ce0. 
I think this test wasn't checked after migration to Combobox.

**Review by poulch — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]
